### ISSUE TITLE
Can't navigate to first tree items via keyboard on website

### ISSUE THREAD

**Issue by GCHQ-Developer-847** [mi6/ic-design-system#1478]
## Summary of the bug
When using the tree view on the v3 website, tabbing to the first item and pressing Enter doesn't do anything; it doesn't take you to that page.

## 🪜 How to reproduce
1. Go to any of the parent pages on the website e.g. [Components](https://mi6.github.io/ic-design-system-githubpages/v3/components).
2. Tab / arrow key down to and select another page e.g. the Accordion page.
3. Tab back onto the Components page tree item and press Enter.
4. See that the tree item shows the selected state but it doesn't actually take you back to that page.

## 🖥 📱 Device
- Browser: Chrome

## 🧐 Expected behaviour
You should be able to navigate to the first item in the tree view just like you can with other tree items.

## Additional info
It happens on every page on the website.

All other tree items seem to work fine when using the keyboard. And there are no problems when using a mouse.
